pkg/gadgets/top/block-io/types/types.go

```diff
@@ -28,23 +28,32 @@ const (
 	TIME
 )
 
-var SortByDefault = []string{"-io", "-bytes", "-us"}
+var SortByDefault = []string{"-ios", "-bytes", "-time"}
 
 // Stats represents the operations performed on a single file
 type Stats struct {
 	eventtypes.CommonData
 
-	Write      bool   `json:"write,omitempty" column:"write"`
+	Pid        int32  `json:"pid,omitempty" column:"pid"`
+	Comm       string `json:"comm,omitempty" column:"comm"`
+	Write      bool   `json:"write,omitempty" column:"r/w,maxWidth:3"`
 	Major      int    `json:"major,omitempty" column:"major"`
 	Minor      int    `json:"minor,omitempty" column:"minor"`
 	Bytes      uint64 `json:"bytes,omitempty" column:"bytes"`
-	MicroSecs  uint64 `json:"us,omitempty" column:"us"`
-	Operations uint32 `json:"io,omitempty" column:"io"`
-	MountNsID  uint64 `json:"mountnsid,omitempty" column:"mountnsid"`
-	Pid        int32  `json:"pid,omitempty" column:"pid"`
-	Comm       string `json:"comm,omitempty" column:"comm"`
+	MicroSecs  uint64 `json:"us,omitempty" column:"time"`
+	Operations uint32 `json:"io,omitempty" column:"ios"`
```

Comment:
In general i think this is a good idea and creates clarity.
What about `Ops` instead of `Operations` which most of the time will get shortend from the columns library to `Oper....`?

Reply:
Right, `ops` looks good.